Add RPG tilesets

drevantonder opened the pull request:
I find that these tiles work very well with tiled.

drevantonder committed:
Add files via upload

drevantonder committed:
Create credits.md

drevantonder committed:
Delete roguelikeSheet_transparent.png

drevantonder committed:
Add files via upload

drevantonder committed:
Add files via upload

drevantonder committed:
Update credits.md

bjorn commented:
First of all thanks for contributing!

To be honest I'm not really sure yet where to go with this repository. There is of course a near infinite amount of tilesets that could potentially be used with Tiled. So we need to decide where to draw the line.

But we also need to start somewhere, and this looks like a good start. A few things to note:
- Please rename `credits.md` to `README.md`, so that its contents will show when browsing to the `RPG` folder.
- Please include a direct link to the source of the set (in this case, that appears to be http://www.kenney.nl/assets/roguelike-rpg-pack).
- Please consider adding `.tsx` external tileset files, that contain the tileset parameters and terrain definitions for the terrain tiles present in `roguelikeSheet_transparent.png`.
- Please consider including a small example map.

The idea is that people checking out these resources will be able to get started quickly, and don't have to figure out all the right settings.

drevantonder committed:
Update and rename credits.md to README.md

drevantonder committed:
Add files via upload

drevantonder committed:
Update README.md

drevantonder committed:
Update README.md

drevantonder committed:
Add files via upload

drevantonder commented:
- I've renamed `credits.md` to `README.md`
- I put the sources in `README.md`
- Ive added `.tsx` files for all the files.
- And I have added a Example Map `sample.tmx`

drevantonder committed:
Update sample.tmx

bjorn commented:
Thanks a lot, looks like a good demonstration now!

I did notice the example map has some tile sets added twice, but that's basically a bug in Tiled so I'll fix that (if I haven't already) and resolve it by re-saving the map.
